### PR TITLE
Add Fuze.app latest

### PR DESCRIPTION
**Pull request by jkorona-r7**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [X] Named the cask according to the [token reference].
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.
- [X] Checked there are no [open pull requests] for the same cask.
- [X] Checked the cask was not already refused in [closed issues].
- [X] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

**Review by gobinathm — APPROVED** [review]
Validated locally and did not see any issue
```
$ brew cask install https://raw.githubusercontent.com/jkorona-r7/homebrew-cask/cbb32815db1bff85c8086fe140f0f9d3a1f0a431/Casks/fuze.rb
==> Downloading https://raw.githubusercontent.com/jkorona-r7/homebrew-cask/cbb32815db1bff85c8086fe140f0f9d3a1f0a431/Casks/fuze.rb
######################################################################## 100.0%
==> Downloading https://citadel.thinkingphones.com/api/v1/applications/fuzeapp/mac/versions/latest/download/installer
######################################################################## 100.0%
==> No checksum defined for Cask fuze, skipping verification
==> Moving App 'Fuze.app' to '/Applications/Fuze.app'
🍺  fuze was successfully installed!
```
```
$ brew cask uninstall fuze
==> Removing App: '/Applications/Fuze.app'
```

**Comment by miccal** [comment]
Ping @vitorgalvao RE https://github.com/caskroom/homebrew-cask/pull/18812/files.

File: Casks/fuze.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'fuze' do
+  version :latest
+  sha256 :no_check
+
+  # citadel.thinkingphones.com was verified as official when first introduced to the cask
+  url 'https://citadel.thinkingphones.com/api/v1/applications/fuzeapp/mac/versions/latest/download/installer'
```
Comment (vitorgalvao) [inline Casks/fuze.rb]: I cannot find this on the website. Where is it?

Comment (jkorona-r7) [inline Casks/fuze.rb]: So this one is a little weird. The download available at [https://www.fuze.com/download](https://www.fuze.com/download) which was used in the previous cask actually downloads an older version of Fuze  (16.x).

If you go to the website and follow the "Customer Portal Login" link in the top right you're taken to the online portal. From there, you can access a help document ([Fuze Desktop GA - download instructions.pdf](https://github.com/caskroom/homebrew-cask/files/874132/Fuze.Desktop.GA.-.download.instructions.pdf)) which provides a link to the latest version (17.x).

Comment (vitorgalvao) [inline Casks/fuze.rb]: Unfortunately, that makes this a [walled build](https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask), so we cannot accept it.

Thank you anyway.

Comment (gobinathm) [inline Casks/fuze.rb]: @vitorgalvao Thats new information for me. Thank you :)

**Pull request closed without merging**